include/exadg/utilities/input_parameters.h

```diff
@@ -0,0 +1,53 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_UTILITIES_INPUT_PARAMETERS_H_
+#define INCLUDE_EXADG_UTILITIES_INPUT_PARAMETERS_H_
+
+// deal.II
+#include <deal.II/base/parameter_handler.h>
+
+namespace ExaDG
+{
+struct InputParameters
+{
+  InputParameters() : input_grid_file(), read_external_grid(false)
+  {
+  }
+
+  void
+  add_parameters(dealii::ParameterHandler & prm, std::string const & subsection_name = "Input")
+  {
+    // clang-format off
+    prm.enter_subsection(subsection_name);
+      prm.add_parameter("GridFile", input_grid_file,    "External input grid file.");
+      prm.add_parameter("ReadGrid", read_external_grid, "Decides whether a grid file is read.");
+    prm.leave_subsection();
+    // clang-format on
+  };
+
+  std::string input_grid_file;
+
+  bool read_external_grid;
```

Comment:
I am not sure if this boolean is really needed. The application, more precisely the function `create_grid()`, will call a certain function that reads a grid from a file, so the parameter `read_external_grid` is somewhat redundant. I consider it a special case that an application can both read in a file or construct the grid internally. In such a special case, the boolean parameter `read_external_grid` should only live inside this particular `Application`, but not be a general parameter in ExaDG in my opinion.

Reply:
Okay, I see your point. During development, having the variable in the input file made sense for me, because I was switching between grids to check them. But yes, I agree it is special case!

Reply:
addressed in ce5731610269cac685066d2a69b823c059586cd3.